app/main.py

```diff
@@ -2,0 +1,23 @@
+import os
+
+
+def move_file(command: str) -> None:
+    parts = command.split(" ")
+
+    source_file, path = parts[1], parts[2]
+
+    if not os.path.isfile(source_file):
+        raise FileNotFoundError(f"Source file '{source_file}' not found.")
+
+    if path.endswith("/"):
+        os.makedirs(path, exist_ok=True)
```

Comment:
Consider handling the case where the directory already exists to avoid potential errors. You can use `os.makedirs(path, exist_ok=True)` to create directories without raising an error if they already exist.